src/plugins/data/public/search/expressions/esdsl.test.ts

```diff
@@ -16,26 +21,38 @@ jest.mock('@kbn/i18n', () => {
   };
 });
 
-jest.mock('../../services', () => ({
-  getUiSettings: () => ({
-    get: () => true,
-  }),
-  getSearchService: () => ({
-    search: jest.fn((params: any) => {
-      return {
-        toPromise: async () => {
-          return { rawResponse: params };
+describe('esdsl', () => {
+  let getStartServices: StartServicesAccessor<DataStartDependencies, DataPublicPluginStart>;
+  let startDependencies: MockedKeys<
+    StartServicesAccessor<DataStartDependencies, DataPublicPluginStart>
+  >;
+  let esdsl: EsdslExpressionFunctionDefinition;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
Is that possible to avoid using that in favor of `mockResolvedValueOnce` in the particular test? In my experience, it can make the test suite brittle?

Reply:
just copy pasted this from another test, i think it makes sense for the tests to look alike and use similar techniques to acomplish something. i agree its not the best and maybe we should at some point refactor some of the tests.

Reply:
Oh, I overlooked that. Makes total sense 👍